### ISSUE TITLE
Packages conflict on Laravel 7

### ISSUE THREAD

**Issue by ilnar-kantug** [tg-bot-api/bot-api-base#37]
## Can't install this package on Laravel 7 instalation

composer require tg-bot-api/bot-api-base

gives error

```
  Problem 1
    - Conclusion: remove phpdocumentor/reflection-common 2.2.0
    - Conclusion: don't install phpdocumentor/reflection-common 2.2.0
    - phpdocumentor/reflection-docblock 4.3.0 requires phpdocumentor/reflection-common ^1.0.0 -> satisfiable by phpdocumentor/reflection-common[1.0, 1.0.1].
    - phpdocumentor/reflection-docblock 4.3.1 requires phpdocumentor/reflection-common ^1.0.0 -> satisfiable by phpdocumentor/reflection-common[1.0, 1.0.1].
    - Can only install one of: phpdocumentor/reflection-common[1.0, 2.2.0].
    - Can only install one of: phpdocumentor/reflection-common[1.0.1, 2.2.0].
    - Installation request for phpdocumentor/reflection-common (locked at 2.2.0) -> satisfiable by phpdocumentor/reflection-common[2.2.0].
    - Installation request for tg-bot-api/bot-api-base ^1.6 -> satisfiable by tg-bot-api/bot-api-base[1.6.0].
    - Conclusion: don't install phpdocumentor/reflection-docblock 5.1.0|install phpdocumentor/reflection-docblock 4.3.0|install phpdocumentor/reflection-docblock 4.3.1
    - Conclusion: remove phpdocumentor/reflection-docblock 5.1.0|install phpdocumentor/reflection-docblock 4.3.0|install phpdocumentor/reflection-docblock 4.3.1
    - tg-bot-api/bot-api-base 1.6.0 requires phpdocumentor/reflection-docblock ^4.3 -> satisfiable by phpdocumentor/reflection-docblock[4.3.0, 4.3.1, 4.3.2, 4.3.3, 4.3.4].
    - Can only install one of: phpdocumentor/reflection-docblock[4.3.2, 5.1.0].
    - Can only install one of: phpdocumentor/reflection-docblock[4.3.3, 5.1.0].
    - Can only install one of: phpdocumentor/reflection-docblock[4.3.4, 5.1.0].
    - Installation request for phpdocumentor/reflection-docblock (locked at 5.1.0) -> satisfiable by phpdocumentor/reflection-docblock[5.1.0].

```
Also tried to install version 1.5.0, but faced the same package conflict error.

* PHP 7.4:
* Laravel v7.18.0:

**Comment by greenplugin** [comment]
Hi! Thanks for the issue. It will be fixed in version 6.1

**Comment by greenplugin** [comment]
Version 1.6.1 has been released. https://packagist.org/packages/tg-bot-api/bot-api-base
